src/rust/examples/slapper/src/main.rs

```diff
@@ -31,6 +40,8 @@ enum Commands {
     Lots 
 }
 
+// Never used function, I am not sure of its purpose - FssAy
+#[allow(dead_code)]
```

Comment:
I think we can just go ahead and remove this function. In. the js cli this is the entry point for color_test, but since Rust works differently this became redundant and I didn't remove it.